NGitLab/Models/BranchProtect.cs

```diff
@@ -25,13 +25,13 @@ public BranchProtect(string branchName)
         public bool AllowForcePush { get; set; } = false;
 
         [JsonPropertyName("allowed_to_merge")]
-        public AccessLevelInfo[] AllowedToMerge { get; set; }
+        public AccessControl[] AllowedToMerge { get; set; }
```

Comment:
Although it make sense to have AccessControl there, it's also a breaking change for BranchProtect. Is this change mandatory for you? Otherwise, I would suggest to keep AccessLevelInfo on BranchProtect at the moment and push ProtectedTag with AccessControl as it's new.

Then, there are multiple fixes we would like to make and also requires breaking changes like https://github.com/ubisoft/NGitLab/issues/491. We could add the AccessControl update in the future rework.

Reply:
This change isn't mandatory for me. I modified `BranchProtect` because introducing the `Id` attribute into `AccessLevelInfo` caused an error in `ProtectBranch_Test`.

I have removed the `Id` attribute from `AccessLevelInfo` and revoked the changes to `BranchProtect`.